src/components/DateRangePicker.jsx

```diff
@@ -268,9 +275,9 @@ export default class DateRangePicker extends React.Component {
   }
 
   isDayAfterHoveredStartDate(day) {
-    const { startDate, endDate } = this.props;
+    const { startDate, endDate, minimumNights } = this.props;
     const { hoverDate } = this.state;
-    return !!startDate && !endDate && isNextDay(hoverDate, day) &&
+    return !!startDate && !endDate && (isNextDay(hoverDate, day) && minimumNights > 0) &&
```

Comment:
You do because I am amending the `isNextDay` condition to be dependent on `minimumNights`. If it's not wrapped and `minimumNights={0}`, then the adjacent dates don't get "hovered".